default.json

```
@@ -107,6 +107,16 @@
       "datasourceTemplate": "docker",
       "versioningTemplate": "docker"
     },
+    {
+      "description": "Update Github Action references in the markdown files",
+      "fileMatch": ["\\.md$"],
+      "matchStrings": [
+        "\\suses: (?<depName>[\\w-]+\\/[\\w-]+)(?<path>\\/.*)?@(?<currentValue>v\\d+\\.\\d+\\.\\d+)",
+        "\\suses: (?<depName>[\\w-]+\\/[\\w-]+)(?<path>\\/.*)?@(?<currentDigest>[a-z0-9]{40}|[a-z0-9]{64}) # renovate: tag=(?<currentValue>v\\d+\\.\\d+\\.\\d+)"
```

```suggestion
        "\\suses: (?<depName>[\\w-]+\\/[\\w-]+)(?<path>\\/.*)?@(?<currentDigest>[a-z0-9]{40}|[a-z0-9]{64}) # tag=(?<currentValue>v\\d+\\.\\d+\\.\\d+)"
```
we now need to support the new default value.

---

Good catch!

You're right, we removed the `renovate:` comment that Renovate bot added when it updated a GitHub Action `.yml` file:

- https://github.com/renovatebot/renovate/pull/14110